specification/cognitiveservices/data-plane/Language/preview/2024-11-15-preview/analyzeconversations-authoring.json

```diff
@@ -1127,6 +1127,256 @@
         }
       }
     },
+    "/authoring/analyze-conversations/projects/{projectName}/exported-models": {
+      "get": {
+        "operationId": "ConversationalAnalysisAuthoring_ListExportedModels",
+        "description": "Lists the exported models belonging to a project.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "projectName",
+            "in": "path",
+            "description": "The new project name.",
+            "required": true,
+            "type": "string",
+            "maxLength": 100
+          },
+          {
+            "$ref": "#/parameters/Azure.Core.TopQueryParameter"
+          },
+          {
+            "$ref": "#/parameters/Azure.Core.SkipQueryParameter"
+          },
+          {
+            "$ref": "#/parameters/Azure.Core.MaxPageSizeQueryParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/PagedConversationalAnalysisAuthoringExportedTrainedModel"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful List Exported Models": {
+            "$ref": "./examples/analyzeconversations-authoring/SuccessfulListExportedModels.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/{projectName}/exported-models/{exportedModelName}": {
+      "get": {
+        "operationId": "ConversationalAnalysisAuthoring_GetExportedModel",
+        "description": "Gets the details of an exported model.",
```

Comment:
Add summary in description instead of remarks.

Reply:
Hi Gouri, Thanks for your comment. I referenced this description from the Swagger specification for the 2024-11-15-preview version here: 
https://dev.azure.com/msazure/Cognitive%20Services/_git/API-Language-Authoring?path=/UnifiedAuthoring.APIs/Swagger/Microsoft.ConversationalLanguageUnderstanding.Authoring.v2024-11-15-preview-github.json&version=GBmain&line=1011&lineEnd=1011&lineStartColumn=24&lineEndColumn=63&lineStyle=plain&_a=contents

Following the rules of typespec conversion of making minimum changes to the original swagger, it might be better to retain the existing description here? Thank you.